unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -714,6 +714,31 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
               Tools for you, the lock manager
             </span>
             <div className="grid gap-4">
+              <Card className="grid grid-cols-1 gap-2 md:items-center md:grid-cols-3">
+                <div className="md:col-span-2">
+                  <Card.Label
+                    title="Manage Attendees"
+                    description="See who is attending your event, invite people with airdrops and more!"
+                  />
+                </div>
+                <div className="md:col-span-1">
+                  {eventLocks?.map(({ lockAddress, network }) => {
+                    return (
+                      <Button
+                        key={lockAddress}
+                        as="a"
```

Comment:
Probably use next link here since it's a relative link

Reply:
Ha! ok let me try!